README.md

```diff
@@ -12,115 +12,131 @@ npx knex migrate:latest
+| name           | purpose                                        |
+| -------------- | ---------------------------------------------- |
+| Login          | View for user to enter their login credentials |
+| Register       | View for user to sign up for the App           |
+| Home           | Placeholder                                    |
+| Create Project | Placeholder                                    |
+| Events         | Placeholder                                    |
 
-  ## Actions (Client Side)
+## Reducers (Client Side)
 
-  | type | data (e.g auth)| purpose |
-  | --- | --- | --- |
-  | CHANGE_PAGE | Placeholder | Enter Purpose |
-  | ADD_USER_INFO | Placeholder | Enter Purpose |
-  | CHANGE_NAV_STATE | Placeholder | Enter Purpose |
-  | SET_EVENTS | Placeholder | Enter Purpose |
+| name         | purpose                                                              |
+| ------------ | -------------------------------------------------------------------- |
+| auth         | Store information regarding user logins, auth status and auth errors |
+| Create User  | Placeholder                                                          |
+| Current Page | Placeholder                                                          |
+| Set Events   | Placeholder                                                          |
 
+## Actions (Client Side)
 
-  ## API (Client - Server)
+| type             | data (e.g auth) | purpose       |
+| ---------------- | --------------- | ------------- |
+| CHANGE_PAGE      | Placeholder     | Enter Purpose |
+| ADD_USER_INFO    | Placeholder     | Enter Purpose |
+| CHANGE_NAV_STATE | Placeholder     | Enter Purpose |
+| SET_EVENTS       | Placeholder     | Enter Purpose |
 
-| Method | Endpoint | Protected | Usage | Response |
-| --- | --- | --- | --- | --- |
-| Post | /api/v1/sendRegistrationEmail | Yes | Send Registration Email | Response Placeholder |
-| Post | /api/v1/sendReminderEmail | Yes | Send Reminder Email | Response Placeholder |
-| Set | application/json | Yes | Accept json Header | Response Placeholder |
-| Send | Placeholder | Yes | Send Email | Response Placeholder |
-| Send | Placeholder | Yes | Send Info | Response Placeholder |
+## API (Client - Server)
 
+| Method | Endpoint                      | Protected | Usage                   | Response             |
+| ------ | ----------------------------- | --------- | ----------------------- | -------------------- |
+| Post   | /api/v1/sendRegistrationEmail | Yes       | Send Registration Email | Response Placeholder |
+| Post   | /api/v1/sendReminderEmail     | Yes       | Send Reminder Email     | Response Placeholder |
+| Set    | application/json              | Yes       | Accept json Header      | Response Placeholder |
+| Send   | Placeholder                   | Yes       | Send Email              | Response Placeholder |
+| Send   | Placeholder                   | Yes       | Send Info               | Response Placeholder |
```

Comment:
❔Might be good to add a technical debt ticket to the backlog to add docblocks to the Express routes, so that we can maintain all of the API info in one place.
💭 Alternatively, we could add an enhancement to the backlog to integrate [Swagger](https://github.com/swagger-api/swagger-node/blob/987b27d5778cedad59f0424a87fbad40bd39653a/docs/introduction.md), so that we can have self-documenting API methods..